core-web/libs/portlets/edit-ema/ui/tsconfig.lib.json

```diff
@@ -8,5 +8,5 @@
         "types": []
     },
     "exclude": ["src/**/*.spec.ts", "src/test-setup.ts", "jest.config.ts", "src/**/*.test.ts"],
-    "include": ["src/**/*.ts"]
+    "include": ["src/**/*.ts", "../../../utils-testing/src/lib/seo-mock.ts"]
```

Comment:
Why?

Reply:
Not sure, I did not change this file manually.